examples/codegen-cluster.yaml

```diff
@@ -0,0 +1,77 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+
+blueprint_name: codegen-cluster
+vars:
+  project_id: PROJECT_ID
```

Comment:
Do not use string literal as a placholder, instead do following:

```yaml
vars:
  project_id: #  you project id
  ...
```

This way toolkit will not attempt to deploy into project "PROJECT_ID"

Reply:
@mr0re1 done, please let me know if anything else is needed